src/main/java/beeboo/components/TimeConverter.java

```diff
@@ -0,0 +1,35 @@
+package beeboo.components;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The TimeConverter class provides functionality to convert a string
+ * representation of date and time into a LocalDateTime object.
+ */
+public class TimeConverter {
+
+    /**
+     * Converts a string representation of a date and time into a LocalDateTime object.
+     * The string should be in the format "yyyy-MM-dd HHmm" or "yyyy-MM-dd".
+     * If only the date is provided, the time is assumed to be 00:00.
+     *
+     * @param date the string representation of the date and time
+     * @return a LocalDateTime object representing the provided date and time
+     * @throws DateTimeParseException if the date string is not in a valid format
+     */
+    public static LocalDateTime timeConverter(String date) throws DateTimeParseException {
```

Comment:
Consider naming this method convertTime instead of timeConverter such that it is a verb instead of a noun.